clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/client/EcsLoadbalancerCacheClient.java

```diff
@@ -56,6 +55,12 @@ public EcsLoadbalancerCacheClient(Cache cacheView, ObjectMapper objectMapper) {
 
   private Set<Map<String, Object>> fetchFromCache(String account, String region) {
     String accountFilter = account != null ? account : "*";
+    if (!accountFilter.equals("*")) {
```

Comment:
nit: reverse the check "*".equals(accountFilter). If the logic for accountFilter changes in the future and it's null, the current implementation would throw an NPE.